src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -78,11 +80,11 @@ public double getY(){
   private double kBaseI = 0.35;
   private double kBaseD = 0;
   //Elbow arm PID values
-  private double kElbowUpP = 5;
+  private double kElbowUpP = 4.5;
```

Comment:
You mentioned yesterday that PID was 5?  Is it 4.5 or 5?

Reply:
At this point, I have no idea
Both values have been used intermittently, both values work fine

Reply:
closing, would be good to check what we ran with during houston district and use that.  Just to be consistent